assets/javascripts/kitten/components/notifications/alert.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import classNames from 'classnames'
+import { CloseButton } from 'kitten/components/buttons/close-button'
+
+export class Alert extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      show: props.show
+    }
+
+    this.handleCloseClick = this.handleCloseClick.bind(this)
+  }
+
+  handleCloseClick() {
+    this.setState({
+      show: false,
+    })
+  }
+
+  renderCloseButton() {
+    if (!this.props.closeButton) return
+
+    return (
+      <CloseButton className="k-ButtonIcon--carbon k-Alert__close"
+                   closeButtonLabel={ this.props.closeButtonLabel }
```

Comment:
Ici on peut peut-être, mettre les `props` dans une `const {}`, pour éviter de remettre `this.prop` devant chaque props .